src/main/java/org/myrobotlab/service/data/Script.java

```diff
@@ -1,58 +1,21 @@
 package org.myrobotlab.service.data;
 
-import java.io.File;
 import java.io.Serializable;
 
 public class Script implements Serializable {

```

Comment:
Probably needs javadoc and probably needs to implement equals() and hashcode(), just to be safe